xml/gl.xml

```diff
@@ -82,6 +82,210 @@ typedef unsigned int GLhandleARB;
     <!-- SECTION: GL parameter class type definitions. -->
 
     <groups>
+        <group name="EvalMapsModeNV">
+            <enum name="GL_FILL_NV">
```

Comment:
These should have end tags I believe.

```suggestion
            <enum name="GL_FILL_NV"/>
```

Reply:
Ah will fix, also it took me roughly 5 hours